src/types/types.ts

```diff
@@ -174,12 +174,16 @@ export type ScanOptions = {
 export type scanJob = {
   id: number;
   report_id: number;
+  status: string;
 };
 
 export type StatusDetails = {
   job_status_message: string;
 };
 
+// TODO: the object returned from the API representing scanJob and MostRecentScan are the same;
+// consider this in a future refactor. Also, MostRecentScan is a ScanJob, not a scan. Yeah, quipucords
+// is confusing...
```

Comment:
optional: may reconsider and remove the personal touch on the TODO

Reply:
sure

...what you had originally worked too... you could have just removed the last sentence